prep-html-links/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Home Page</title>
+  </head>
+  <body>
+    <h1>Home page</h1>
+    <a
+      href="file:///C:/Users/malsy/repos/prep-1121-code-solutions/prep-html-links/contact-us/contact-us.html"
+      >Click here to go to the contact page
+      <img src="contact-us/contact-us.html"
+    /></a>
```

Comment:
You can leave out this `<img>`, we just want to add `href` attributes to our anchor tags.

Reply:
Ahh ok I get it now. I can use the format for a relative path shown in the <img>tag section on w3s, inside of the <a> tag. Whereas I was trying to implement both <a> tag and <img>. Thanks for the comments!

Reply:
Ahh now I get it! I should use what I learned from the img tag reading about the relative url, and implement that in my anchor tags. Thanks for the comments!